config/checkstyle_checks.xml

```diff
@@ -147,6 +147,7 @@
       <property name="tokens" value="ANNOTATION_FIELD_DEF"/>
       <property name="tokens" value="PACKAGE_DEF"/>
       <property name="tokens" value="ENUM_CONSTANT_DEF"/>
+      <property name="tokens" value="VARIABLE_DEF"/>
```

Comment:
@Vampire I don't see anywhere in issue saying we are expecting this change here.

Reply:
Hm, I don't remember why I added it, whether I just followed the "we adhere to our own rules" mantra of checkstyle or whether someone said I should add it.
Do you think it does not make sense to have it? Then I can of course also remove it.

Reply:
I will let @romani decide since he has decided most of these rules. This is why I added it as a comment instead of a changes requested. I don't have an issue with it either way.

Reply:
yes, we activate all on our own code. If for some reason we do not like behavior, it is good sign that other might not like it either, so it might require better design or we put a wide comment why such validation is not for us.

In this particular case, it does not trigger any code changes, so lets keep it active, to alarm us when smth will go in different way. We can always change config to smth more suitable for us.